OrbitCore/SamplingProfiler.h

```diff
@@ -137,6 +139,8 @@ class SamplingProfiler {
   void SortByThreadUsage();
   void ProcessSamples();
   void UpdateAddressInfo(uint64_t address);
+  ThreadSampleData& GetSummary() { return m_ThreadSampleData[0]; }
+  [[nodiscard]] unsigned int GetCountOfFunction(uint64_t function_address);
```

Comment:
GetCountOfFunction can be declared const, can't it? At least when GetSummary is declared const.

Reply:
Tried to make GetSummary const. So could made this also const.